src/python/WMArchive/Service/Data.py

```diff
@@ -29,6 +29,14 @@
 from WMArchive.Service.Manager import WMArchiveManager
 from WMArchive.Utils.Regexp import PAT_UID, PAT_QUERY, PAT_INFO
 
+def results(result):
+    "Return results as a list data type. Set proper status in case of failures"
+    if 'status' in result and 'Not supported' in result['status']:
+        cherrypy.response.status = "406 Not Acceptable"
```

Comment:
Done in https://github.com/dmwm/WMArchive/pull/101
On  0, ticoann <notifications@github.com> wrote:
> > @@ -29,6 +29,14 @@
> >  from WMArchive.Service.Manager import WMArchiveManager
> >  from WMArchive.Utils.Regexp import PAT_UID, PAT_QUERY, PAT_INFO
> >  
> > +def results(result):
> > +    "Return results as a list data type. Set proper status in case of failures"
> > +    if 'status' in result and 'Not supported' in result['status']:
> > +        cherrypy.response.status = "406 Not Acceptable"
> 
> Although this is fine but could you just set it as integer status code? That is what generally expected all other status in WMCore code.
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/dmwm/WMArchive/pull/98/files#r52220655